src/libs/IOUUtils.js

```diff
@@ -28,6 +28,41 @@ function calculateAmount(numberOfParticipants, total, currency, isDefaultUser =
     return Math.round((finalAmount * 100) / currencyUnit);
 }
 
+/**
+ * Edits the IOU total when a transaction is edited
+ *
+ * @param {Object} iouReport
+ * @param {Object} initialTransaction
+ * @param {Object} updatedTransaction
+ * @param {Boolean} isExpenseReport
+ * @returns {Object}
+ */
+function editIOUTotal(iouReport, initialTransaction, updatedTransaction, isExpenseReport = false) {
```

Comment:
@mananjadhav Just saw the guidelines and wanted to confirm:
This can surely be replaced by using `updateIOUOwnerAndTotal` twice (once to remove the old amount and then to add the new amount) but thought of adding a utils component to make it look better, if DRY is more important then I can remove this, thanks!

Reply:
I checked and I think it makes sense to use `updateIOUOwnerAndTotal` from your current L48 onwards. Can we please update that? We can keep the rest of the method as is, and agreed on keeping it in `IOUUtils.`

Reply:
Done!